static/js/trips-map.js

```diff
@@ -1,81 +1,89 @@
-mapboxgl.accessToken = 'pk.eyJ1IjoibmVvbWFkIiwiYSI6ImNqMHRrZ3ZwdzAwNDgzMm1kcHRhMDdsZGIifQ.bOSlLkmc1LBv0xAbcZXpog'
-var map = new mapboxgl.Map({
-  container: 'map',
-  style: 'mapbox://styles/mapbox/streets-v9',
-  center: [-10, 45],
-  zoom: 2
+// init Map
+const accessToken = 'pk.eyJ1IjoibmVvbWFkIiwiYSI6ImNqMHRrZ3ZwdzAwNDgzMm1kcHRhMDdsZGIifQ.bOSlLkmc1LBv0xAbcZXpog'
+const map = L.map('map', {
+  center: [0, 0],
+  zoom: 2,
+  minZoom: 2,
+  zoomControl: false,
+  layers: L.tileLayer(`https://api.mapbox.com/styles/v1/mapbox/light-v9/tiles/256/{z}/{x}/{y}?${''}access_token=${accessToken}`)
 })
 
-// Init array to create lines between marker 
-const lineCoords = []
+new L.Control.Zoom({position: 'topright'}).addTo(map)
+const localizeUser = L.control.locate({position: 'topright', locateOptions: {maxZoom: 14}}).addTo(map)
 
-// Previous trips
-locations.forEach(point => {
-  const popup = new mapboxgl.Popup({offset: [10, 0]})
-      .setText(point.date)
-
-  const el = document.createElement('div')
-  el.classList.add('marker')
+// create icon model
+const iconOptions = {
+  iconSize: [30, 50],
+  iconAnchor: [15, 42],
+  popupAnchor: [0, -40],
+  viewBox: '300 100 1200 1200',
+  path: 'm 768,896 q 0,106 -75,181 -75,75 -181,75 -106,0 -181,-75 -75,-75 -75,-181 0,-106 75,-181 75,-75 181,-75 106,0 181,75 75,75 75,181 z m 256,0 q 0,-109 -33, -179 L 627,-57 q -16,-33 -47.5,-52 -31.5,-19 -67.5,-19 -36,0 -67.5,19 Q 413,-90 398, -57 L 33,717 Q 0,787 0,896 q 0,212 150,362 150,150 362,150 212,0 362,-150 150,-150 150,-362 z',
+  transform: 'matrix(1,0,0,-1,364.47458,1270.2373)'
+}
 
-  lineCoords.push(point.position)
+class Icon extends L.Icon {
```

Comment:
seems like copy and paste. We'll need to refactor this code into one single spot